app/components/user-menu.tsx

```diff
@@ -0,0 +1,134 @@
+import clsx from "clsx";
+import React from "react";
+import { useFetcher, useLoaderData, useNavigate } from "react-router";
+import type { loader } from "~/routes/_index";
+import { CopyButton } from "./copy-button";
+import { Input } from "./input";
+
+function UserMenu() {
+	const { user } = useLoaderData<typeof loader>();
+	const navigate = useNavigate();
+	const [showInvite, setShowInvite] = React.useState(false);
+	const fetcher = useFetcher();
+
+	const fetched = React.useRef(false);
+
+	React.useEffect(() => {
+		if (showInvite && !fetched.current && fetcher.state === "idle") {
+			fetcher.load("/invite");
+			fetched.current = true;
+		}
+
+		if (!showInvite) {
+			fetched.current = false;
+		}
+	}, [showInvite, fetcher.state]);
```

Comment:
missing fetcher.load


```suggestion
	}, [showInvite, fetcher.state, fetcher.load]);
```